rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -443,15 +442,15 @@ object Runtime {
                   ListParWithRandom,
                   ListParWithRandom,
                   TaggedContinuation
-                ](dataDir, mapSize, Branch.MASTER)
+                ](context, Branch.MASTER)
```

Comment:
Why was this temporary change necessary? Could you not just remove the first change to `Runtime`?